src/Duende.IdentityServer.Demo.csproj

```diff
@@ -11,4 +11,9 @@
 		<PackageReference Include="Serilog.AspNetCore" Version="9.0.0" />
 	</ItemGroup>
 	
+	<Target Name="Parcel" BeforeTargets="Build">
+		<Exec Command="npm install" Condition="'$(Configuration)' == 'Release'"/>
+		<Exec Command="npm run build" />
+	</Target>
```

Comment:
```suggestion
	<Target Name="Parcel" BeforeTargets="Build">
		<Exec Command="node --version" ContinueOnError="true">
			<Output TaskParameter="ExitCode" PropertyName="ErrorCode" />
		</Exec>
		<Error Condition="'$(ErrorCode)' != '0'" Text="Node.js is required to build and run this project. To continue, please install Node.js from https://nodejs.org/, and then restart your command prompt or IDE." />
		<Message Importance="high" Text="Restoring dependencies using 'npm'. This may take several minutes..." />
		<Exec Command="npm install" />
		<Exec Command="npm run build" />
	</Target>
```